### PR TITLE
Update and refactor unit tests

### PR DESCRIPTION
**Pull request by vramperez**

### CODE REVIEW

File: test/unit/APIServer_spec.js [hunk 1]
```diff
@@ -135,29 +115,35 @@ describe('Testing APIServer', function() {
...
 
-    describe('Function "checkUrl"', function() {
+    describe('Function "checkURL"', function() {
 
-        var testCheckUrl = function (body, addTokenErr, checkPathErr, checkResult, done) {
+        var testcheckURL = function (body, userId, getAdminsErr, admins, token, addTokenErr, done) {
 
-            var token = data.DEFAULT_TOKEN;
             var path = '/path';
             var espias;
```
Comment (fdelavega) [inline test/unit/APIServer_spec.js]: Spanish var name

File: test/unit/APIServer_spec.js [hunk 2]
```diff
@@ -190,38 +176,43 @@ describe('Testing APIServer', function() {
 
                 espias = spies;
```
Comment (fdelavega) [inline test/unit/APIServer_spec.js]: You are not using this var in anywhere

File: test/unit/db_sqlite_spec.js [hunk 3]
```diff
@@ -976,7 +978,7 @@ describe('Testing SQLITE database', function () {
 
             var db = getDb(implementations);
 
-            db.getApiKeys(data.DEFAULT_USER, function (err, res) {
+            db.getApiKeys(data.DEFAULT_USER_ID_ID, function (err, res) {
```
Comment (fdelavega) [inline test/unit/db_sqlite_spec.js]: Typo DEFAULT_USER_ID

File: test/unit/db_sqlite_spec.js [hunk 4]
```diff
@@ -960,7 +962,7 @@ describe('Testing SQLITE database', function () {
         var sentence = 'SELECT apiKey, productId, orderId \
             FROM accounting \
             WHERE customer=$user';
-        var params = {'$user': data.DEFAULT_USER};
+        var params = {'$user': data.DEFAULT_USER_ID_ID};
```
Comment (fdelavega) [inline test/unit/db_sqlite_spec.js]: Typo DEFAULT_USER_ID

**Comment by vramperez** [comment]
New commit added

**Pull request merged**